Tests/scripts/test_modeling_rule_report.py

```diff
@@ -121,3 +125,20 @@ def calculate_test_modeling_rule_results(test_modeling_rules_results_files: dict
                     jira_tickets_for_modeling_rule[summary] = sorted_issues_matching_summary[0]
 
     return modeling_rules_to_test_suite, jira_tickets_for_modeling_rule, server_versions
+
+
+def write_test_modeling_rule_to_jira_mapping(artifacts_path: Path, jira_tickets_for_modeling_rule: dict[str, Issue]):
+    test_modeling_rule_to_jira_mapping_file = artifacts_path / TEST_MODELING_RULES_TO_JIRA_MAPPING
+    logging.info(f"Writing test modeling rules to Jira mapping to {test_modeling_rule_to_jira_mapping_file}")
+    with open(test_modeling_rule_to_jira_mapping_file, "w") as test_modeling_rule_to_jira_mapping_fp:
+        test_modeling_rule_to_jira_mapping = {modeling_rule: jira_ticket_to_json_data(jira_ticket)
+                                              for modeling_rule, jira_ticket in jira_tickets_for_modeling_rule.items()}
+        test_modeling_rule_to_jira_mapping_fp.write(json.dumps(test_modeling_rule_to_jira_mapping, indent=4, sort_keys=True,
+                                                               default=str))
+
+
+def read_test_modeling_rule_to_jira_mapping(artifacts_path: Path) -> dict[str, dict[str, str]]:
+    with (contextlib.suppress(Exception),
```

Comment:
Or `info` You decide.
```suggestion
    logging.debug(f"Reading test_modeling_rules_to_jira_mapping from {TEST_MODELING_RULES_TO_JIRA_MAPPING}")
    with (contextlib.suppress(Exception),
```